pkgs/top-level/all-packages.nix

```diff
@@ -5322,6 +5322,13 @@ with pkgs;
   gcc = gcc6;
   gcc-unwrapped = gcc.cc;
 
+  gccStdenv = if (!stdenv.isDarwin) then stdenv else stdenv.override {
+    allowedRequisites = null;
+    cc = gcc;
+    # Include unwrapped binaries like AS, etc. and remove libcxx/libcxxabi
```

Comment:
Wouldn't `as` and friends be from binutils?

Reply:
It's part of cctools, but ar/as are indeed included in binutils.

Reply:
@copumpkin Any idea why this happens when clang isn't available?

```
/nix/store/k3wk9jrs5jxzd0al882034qnd7hgl61m-gcc-wrapper-6.4.0/bin/as: assembler (clang) not installed
```

Reply:
No clue, sorry. Maybe cctools somehow assumes it's clang nowadays? Never looked into running it with gcc

Reply:
Add a `-Q` flag so it uses binutils as instead.